aiosql/adapters/aiosqlite.py

```diff
@@ -37,6 +37,17 @@ async def select_one(conn, _query_name, sql, parameters, record_class=None):
                 result = record_class(**dict(zip(column_names, result)))
         return result
 
+    @staticmethod
+    async def select_value(conn, _query_name, sql, parameters, record_class=None):
+        async with conn.execute(sql, parameters) as cur:
+            result = await cur.fetchone()
+            if record_class is None:
+                result = result[0] if result else None
+            elif result is not None:
+                column_names = [c[0] for c in cur.description]
+                result = record_class(**dict(zip(column_names, result)))
```

Comment:
@wagnerflo I'm having a hard time imagining a scenario where I want a record class off a single value. What do you think about this?

I don't think this case is covered by your tests. Perhaps trying to cover it might help us understand if/when a person would want this and if we should support it. Would you be willing to try that out and see how it feels?

Reply:
Me, too. I just wanted it to theoretically work. Even if a one-value class seems useless (but maybe it isn't) not supporting it would be an inconsistency.

I'll leave the decision up to you but would prefer to write tests only if you decide to leave it supported. Not so much time on my hands currently.

Reply:
Makes sense. Thanks for the reply, I'll think about this a bit more.